src/celeritas/optical/WavelengthShiftData.hh

```diff
@@ -0,0 +1,80 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/WavelengthShiftData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/data/Collection.hh"
+#include "celeritas/Types.hh"
+#include "celeritas/grid/GenericGridData.hh"
+
+namespace celeritas
+{
+namespace optical
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Material dependent scalar property of wavelength shift (WLS).
+ */
+struct ScalarPropertyWLS
+{
+    real_type mean_num_photons{};  //!< Mean number of reemitted photons
+    real_type time_constant{};  //!< Time delay of WLS // units::nanosecond
```

Comment:
This (and anything else that's not internal to a class or wrapped in a `Quantity`) should be in native units:
```suggestion
    real_type time_constant{};  //!< Time delay of WLS [time]
```
From the test code and setup it looks like the units are correctly native, it's just the doc here that needs fixigin.

Reply:
Updated.